ssl/ssl_locl.h

```diff
@@ -790,8 +790,11 @@ struct ssl_ctx_st {
     /* used if SSL's info_callback is NULL */
     void (*info_callback) (const SSL *ssl, int type, int val);
 
-    /* what we put in client cert requests */
-    STACK_OF(X509_NAME) *client_CA;
+    /*
+     * what we put in certificate_authorities extension for TLS 1.3
```

Comment:
Should we note explicitly that this is for both ClientHello and CertifciateRequest?
(Also, if we're ending with a full stop, we should probably start the sentence with a capital letter.)